x-pack/plugins/observability_solution/dataset_quality/public/state_machines/dataset_quality_controller/src/state_machine.ts

```diff
@@ -354,7 +357,21 @@ export const createPureDatasetQualityControllerStateMachine = (
                           actions: ['storeFlyoutOptions'],
                         },
                         BREAKDOWN_FIELD_CHANGE: {
-                          actions: ['storeFlyoutOptions'],
+                          target: 'assertBreakdownFieldIsEcs',
+                        },
+                      },
+                    },
+                    assertBreakdownFieldIsEcs: {
```

Comment:
See if it makes sense to not make it part of the state, as it's only needed to report telemetry. See other [comment](https://github.com/elastic/kibana/pull/188284/files#diff-3b0e093d52c98a6e41ef467440f104bded7b54bfe1da6c78410e68a5938b7696R66).